controllers/storagecluster/cephcluster.go

```diff
@@ -53,6 +54,8 @@ const (
 const (
 	poolTypeData     = "data"
 	poolTypeMetadata = "metadata"
+
+	keyRotationCounterEnvVarName = "KEY_ROTATION_COUNTER"
```

Comment:
Maybe a better name like: CEPH_USER_CURR_GEN

Reply:
Lets hear from others, I believe we can come up with something better.

Reply:
@iamniting This is less about rotation and about identifying the desired generation (the same term we use everywhere, even in rook APIs). So I agree with @rewantsoni but will offer a small change:
`DESIRED_CEPH_USER_GEN`